content/en/docs/tutorials/cluster-management/kubelet-standalone.md

```diff
@@ -0,0 +1,494 @@
+
+To disable swap temporarily:
+
+```shell
+sudo swapoff -a
+```
+
+To make this change persistent across reboots:
+
+Make sure swap is disabled in either `/etc/fstab` or `systemd.swap`, depending how it was
+configured on your system.
+
+### Enable IPv4 packet forwarding
+
+To check if IPv4 packet forwarding is enabled:
+
+```shell
+cat /proc/sys/net/ipv4/ip_forward
+```
+
+If the output is `1`, it is already enabled. If the output is `0`, then follow next steps.
+
+To enable IPv4 packet forwarding, create a configuration file that sets the
+`net.ipv4.ip_forward` parameter to `1`:
+
+```shell
+sudo tee /etc/sysctl.d/k8s.conf <<EOF
+net.ipv4.ip_forward = 1
+EOF
+```
+Apply the changes to the system:
+
+```shell
+sudo sysctl --system
+```
+
+The output is similar to:
+
+```
+...
+* Applying /etc/sysctl.d/k8s.conf ...
+net.ipv4.ip_forward = 1
+* Applying /etc/sysctl.conf ...
+```
+
+## Download, install, and configure the components
+
+{{% thirdparty-content %}}
+
+### Install a container runtime {#container-runtime}
+
+Download the latest available versions of the required packages (recommended).
+
+This tutorial suggests installing the [CRI-O container runtime](https://github.com/cri-o/cri-o)
+(external link).
+
+There are several [ways to install](https://github.com/cri-o/cri-o/blob/main/install.md)
+the CRI-O container runtime, depending on your particular Linux distribution. Although
+CRI-O recommends using either `deb` or `rpm` packages, this tutorial uses the
+_static binary bundle_ script of the
+[CRI-O Packaging project](https://github.com/cri-o/packaging/blob/main/README.md),
+both to streamline the overall process, and to remain distribution agnostic.
+
+The script installs and configures additional required software, such as
+[`cni-plugins`](https://github.com/containernetworking/cni), for container
+networking, and [`crun`](https://github.com/containers/crun) and
+[`runc`](https://github.com/opencontainers/runc), for running containers.
+
+The script will automatically detect your system's processor architecture
+(`amd64` or `arm64`) and select and install the latest versions of the software packages.
+
+**CRI-O**
+
+Visit the [releases](https://github.com/cri-o/cri-o/releases) page (external link).
+
+Download the static binary bundle script:
+
+```shell
+curl https://raw.githubusercontent.com/cri-o/packaging/main/get > crio-install
+```
+
+Run the installer script:
+
+```shell
+sudo bash crio-install
+```
+
+Enable and start the `crio` service:
+
+```shell
+sudo systemctl daemon-reload
+sudo systemctl enable --now crio.service
+```
+
+Quick test:
+
+```shell
+sudo systemctl is-active crio.service
+```
+
+The output is similar to:
+
+```
+active
+```
+
+Detailed service check:
+
+```shell
+sudo journalctl -f -u crio.service
+```
+
+Note: The `cri-o` installer also provides the `crictl` command to manage and inspect
+Pods, containers, and container images. The `crictl` command is out of the scope
+of this tutorial.
```

Comment:
We have a special shortcode for notes
```suggestion
{{< note >}}
The `cri-o` installer also provides the `crictl` command to manage and inspect
Pods, containers, and container images. The `crictl` command is out of the scope
of this tutorial.
{{< /note >}}
```

Reply:
I'll just remove the paragraph, it just adds unnecessary noise and serves no purpose in the tutorial.